website/i18n/de/docusaurus-plugin-content-docs/current/BambooIntegration.md

```diff
@@ -40,46 +40,45 @@ module.exports = {
+Die Berichte sind für alle Frameworks ähnlich und Sie können jeden verwenden: Mocha, Jasmine oder Cucumber.
 
-By this time, we believe you have the tests written up and results are generated in `./testresults/` folder, and your Bamboo is up and running.
+Wir glauben, dass Sie zu diesem Zeitpunkt die Tests geschrieben haben und die Ergebnisse im Ordner `./testresults/` generiert werden und Ihr Bamboo betriebsbereit ist.
 
-## Integrate your tests in Bamboo
+## Integrieren Sie Ihre Tests in Bamboo
 
-1. Open your Bamboo project
+1. Öffnen Sie Ihr Bamboo-Projekt
 
-    > Create a new plan, link your repository (make sure it always points to newest version of your repository) and create your stages
+    > Erstellen Sie einen neuen Plan, verknüpfen Sie Ihr Repository (stellen Sie sicher, dass es immer auf die neueste Version Ihres Repositorys verweist) und erstellen Sie Ihre Stufen
 
-    ![Plan Details](/img/bamboo/plancreation.png "Plan Details")
+    ![Plan-Details](/img/bamboo/plancreation.png "Plan-Details")
 
-    I will go with the default stage and job. In your case, you can create your own stages and jobs
+    Ich werde mit der Standardstufe und dem Standardjob fortfahren. In Ihrem Fall können Sie Ihre eigenen Stufen und Jobs erstellen
 
-    ![Default Stage](/img/bamboo/defaultstage.png "Default Stage")
-2. Open your testing job and create tasks to run your tests in Bamboo
-> **Task 1:** Source Code Checkout
+    ![Standardstufe](/img/bamboo/defaultstage.png "Standardstufe")
+2. Open your testing job and create tasks to run your tests in Bamboo > **Task 1:** Source Code Checkout
+> **Aufgabe 1:** Quellcode-Checkout
+> **Aufgabe 2:** Führen Sie Ihre Tests durch `npm i && npm führen Sie den Test durch`. Sie können *Script* Task und *Shell Interpreter* verwenden, um die obigen Befehle auszuführen (Dadurch werden die Testergebnisse generiert und im Ordner `./testresults/` gespeichert).
 
-> **Task 1:** Source Code Checkout **Task 2:** Run your tests `npm i && npm run test`. You can use *Script* task and *Shell Interpreter* to run the above commands (This will generate the test results and save them in `./testresults/` folder)
+    ![Testlauf](/img/bamboo/testrun.png "Testlauf")
+> **Aufgabe: 3** Fügen Sie *jUnit Parser* Aufgabe hinzu, um Ihre gespeicherten Testergebnisse zu parsen. Bitte geben Sie hier das Verzeichnis der Testergebnisse an (Sie können auch Muster im Ant-Stil verwenden)
 
-    ![Test Run](/img/bamboo/testrun.png "Test Run")
-> **Task: 3** Add *jUnit Parser* task to parse your saved test results. Please specify the test results directory here (you can use Ant style patterns as well)
+    ![jUnit-Parser](/img/bamboo/junitparser.png "jUnit-Parser")
 
-    ![jUnit Parser](/img/bamboo/junitparser.png "jUnit Parser")
+    !\[jUnit Parser\](/img/bamboo/junitparser.png "jUnit Parser") Note: *Make sure you are keeping the results parser task in *Final* section, so that it always get executed even if your test task is failed*
```

Comment:
```suggestion
    ![jUnit Parser](/img/bamboo/junitparser.png "jUnit Parser") Note: *Make sure you are keeping the results parser task in *Final* section, so that it always get executed even if your test task is failed*
```